integration-tests/dotnet/default_instrumentation_dotnet_env_variables.json

```diff
@@ -0,0 +1,9 @@
+{
+  "OTEL_TRACES_SAMPLER_ARG": "endpoint=http://cloudwatch-agent.amazon-cloudwatch:2000",
+  "OTEL_TRACES_SAMPLER" : "xray",
+  "OTEL_EXPORTER_OTLP_PROTOCOL": "http/protobuf",
+  "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT": "http://cloudwatch-agent.amazon-cloudwatch:4316/v1/traces",
+  "OTEL_METRICS_EXPORTER": "none",
+  "OTEL_DOTNET_DISTRO": "aws_distro",
+  "OTEL_LOGS_EXPORTER": "none"
```

Comment:
we want to include more variables than these for dotnet. We want everything here: https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/pkg/instrumentation/defaultinstrumentation_test.go#L72

There might be more as well such as 
```
CORECLR_PROFILER_PATH:                           /otel-auto-instrumentation-dotnet/linux-x64/OpenTelemetry.AutoInstrumentation.Native.so
      DOTNET_STARTUP_HOOKS:                            /otel-auto-instrumentation-dotnet/net/OpenTelemetry.AutoInstrumentation.StartupHook.dll
      DOTNET_ADDITIONAL_DEPS:                          /otel-auto-instrumentation-dotnet/AdditionalDeps
      OTEL_DOTNET_AUTO_HOME:                           /otel-auto-instrumentation-dotnet
      DOTNET_SHARED_STORE:                             /otel-auto-instrumentation-dotnet/store
```

But I'd probably cross reference with what you are currently seeing in your pod to ensure these are accurate

Reply:
Oh wait yes your right.  "CORECLR_ENABLE_PROFILING": "1" is also missing it seems. Just added all of these